openerp/tools/mail.py

```diff
@@ -123,6 +130,30 @@ def html_sanitize(src, silent=True, strict=False, strip_style=False, strip_class
     return cleaned
 
 
+_style_whitelist = ['width', 'height', 'font-size']
+_style_replace = {
+    ''
+}
+
+def sanitize_attributes(html_root):
```

Comment:
Wouldn't it make sense to add this to lxml.html.clean.Cleaner (and possibly even upstream that support)? We're already subclassing it so it's not a huge deal.